app/gameengine.py

```diff
@@ -134,6 +138,7 @@ class Condition:
     Condition_PerformerIsSpecificId = "performer_is_specific_id"
     Condition_PerformerHasAnyTag = "performer_has_any_tag"
     Condition_PlayedSupportThisTurn = "played_support_this_turn"
+    Condition_RevealedCardsHasSameType = "revealed_cards_has_same_type"
```

Comment:
"has" instead of "have" is going to bother me, haha....
Maybe revealed_cards_are_same_type?